torch/tensor.py

```diff
@@ -0,0 +1,394 @@
+import sys
+import torch
+import torch._C as _C
+from collections import OrderedDict
+import torch.utils.hooks as hooks
+import warnings
+import weakref
+from torch._six import imap
+from torch._C import _add_docstr
+
+
+class Tensor(torch._C._TensorBase):
+    r"""Wraps a tensor and records the operations applied to it.
+
+    Variable is a thin wrapper around a Tensor object, that also holds
```

Comment:
nit: Might be worth rewriting this part removing references to `Variable`.

Reply:
I'll delete it for now. I'm not sure how much of the docs should be here or in tensor.rst. We're going to update the documentation after we fix some other the other issues in #5673